apps/polls/views.py

```diff
@@ -55,8 +35,18 @@ def get_success_url(self):
             'dashboard-project-list',
             kwargs={'organisation_slug': self.organisation.slug, })
 
+    def get_or_create_poll(self):
```

Comment:
i can't assure the creator will be the same, as there can exist multiple admins. different admins can access the poll and it may only be created once for each model.